res/values/strings.xml

```diff
@@ -722,6 +722,9 @@
     <string name="emoji_symbols">Symbols</string>
     <string name="emoji_flags">Flags</string>
 
+    <!-- Reactions -->
+    <string name="reactions_details_title">Reactions</string>
```

Comment:
just call it `reactions` - if we need to become more specific, we do that as needed